testcontainers/arangodb.py

```diff
@@ -0,0 +1,64 @@
+"""
+ArangoDB container support.
+"""
+from os import environ
+from testcontainers.core.generic import DbContainer
+from testcontainers.core.waiting_utils import wait_for_logs
+
+
+class ArangoDbContainer(DbContainer):
+    """
+    ArangoDB container.
+
+    Example
+    -------
+    The example will spin up a ArangoDB container.
+    You may use the :code:`get_connection_url()` method which returns a arangoclient-compatible url
+    in format :code:`scheme://host:port`. As of now, only a single host supported (over HTTP).
+    ::
+
+        with ArangoContainer("arangodb:3.9.1") as arango:
+            client = ArangoClient(hosts=arango.get_connection_url())
+
+            # Connect
+            sys_db = arango_client.db(username='root', password='')
+
+            # Create a new database named "test".
+            sys_db.create_database("test")
+    """
+    def __init__(self, image="arangodb:latest", **kwargs):
```

Comment:
```suggestion
    def __init__(self, image="arangodb:latest", port_to_expose=8529, **kwargs):
```

Reply:
This makes it easier to modify the exposed port if necessary (we should also change that on a bunch of other containers at some point).